polling_stations/templates/postcode_view.html

```diff
@@ -5,7 +5,11 @@
 {% load pipeline %}
 
 {% block extra_page_css %}
-{% stylesheet 'map' %}
+
+{% if we_know_where_you_should_vote and station.location %}
```

Comment:
Lovely! Really only loading the JS if we need it.

Not for now, but I wonder if the view should give this logic to the template with something like `can_show_map` in the context? Anyway, this is great!